Dockerfile

```diff
@@ -3,6 +3,20 @@ FROM bitwalker/alpine-elixir:1.4.2
 MAINTAINER PRX <sysadmin@prx.org>
 LABEL org.prx.app="yes"
 
+# install git, aws-cli
+RUN apk update && apk --update add \
+    ca-certificates \
+    git
+
+RUN apk --no-cache update && \
+    apk --no-cache add python py-pip py-setuptools ca-certificates groff less && \
+    pip --no-cache-dir install awscli && \
+    rm -rf /var/cache/apk/*
+
+# install PRX aws-secrets scripts
+RUN git clone -o github https://github.com/PRX/aws-secrets
```

Comment:
Eventually, since we'll be doing this a lot, may be worth adding a script to `aws-secrets`.

```
ADD https://github.com/PRX/aws-secrets/bootstrap_alpine.sh /bootstrap_alpine.sh
RUN sh /bootstrap_alpine.sh
```

Or something.